tests/recipes/test_full_finetune_distributed.py

```diff
@@ -169,6 +175,12 @@ def test_training_state_on_resume(
         runpy.run_path(TUNE_PATH, run_name="__main__")
 
         # Resume training
+        epoch_folder = get_largest_iter_folder(tmpdir)
+        epoch_folder_minus_one = f"epoch_{int(epoch_folder.split('_')[-1]) - 1}"
+        suffix = ".safetensors" if ckpt_type == "hf" else ".bin"
+        model_ckpt_fname = (
+            SHARD_FNAME.format(cpt_idx="1".zfill(5), num_shards="1".zfill(5)) + suffix
+        )
```

Comment:
test changes are related to finding the files. Before they were hardcoded.

Now we retrieve the epoch folder, get the suffix based on ckpt_type, and create the ckpt_name based on the defined standard.